cabal-install/tests/IntegrationTests/exec/should_run/configures_cabal_to_use_sandbox.sh

```diff
@@ -0,0 +1,14 @@
+. ../common.sh
+
+cabal sandbox delete > /dev/null
```

Comment:
These `> /dev/null` redirections are kinda ugly, can we get rid of them at some point?